app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/YoutubeVideoViewHolder.java

```diff
@@ -0,0 +1,67 @@
+package org.fossasia.susi.ai.chat.adapters.viewholders;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.ImageView;
+
+import com.squareup.picasso.Picasso;
+
+import org.fossasia.susi.ai.R;
+import org.fossasia.susi.ai.chat.ChatActivity;
+import org.fossasia.susi.ai.chat.IYoutubeVid;
+import org.fossasia.susi.ai.chat.YoutubeVid;
+import org.fossasia.susi.ai.data.model.ChatMessage;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import timber.log.Timber;
+
+
+/**
+ * @author mohitkumar
+ * <p>
+ * This viewholder is used to display the youtube video thumbnails
+ */
+
+public class YoutubeVideoViewHolder extends MessageViewHolder {
+
+    @BindView(R.id.youtube_view)
+    public ImageView playerView;
+    @BindView(R.id.play_video)
+    public ImageView playBtn;
+    protected ChatMessage model;
+    protected Context currContext;
+    private String videoId;
+    private IYoutubeVid youtubeVid;
+
+    public YoutubeVideoViewHolder(View view, ClickListener clickListener) {
+        super(view, clickListener);
+        ButterKnife.bind(this, view);
+    }
+
+    public void setPlayerView(ChatMessage model, final Context currContext) {
```

Comment:
Context isn't needed, View already has context

Reply:
@iamareebjamal the context here is passed from the ChatFeedRecyclerAdaper and similar to other views the context parameter was included here, so I'll remove the global context object in this viewholder as it is not used anywhere.

Reply:
The parameter is unnecessary. Why do you need it? You have view reference and it has context